.github/workflows/setup.yml

```diff
@@ -6,18 +6,18 @@ on:
       enable_build_jobs:
         description: Whether to enable build jobs.
         value: ${{ jobs.setup.outputs.enable_build_jobs }}
-      build_linux_amdgpu_families:
-        description: Linux families to run builds on.
-        value: ${{ jobs.setup.outputs.build_linux_amdgpu_families }}
-      build_windows_amdgpu_families:
-        description: Windows families to run builds for.
-        value: ${{ jobs.setup.outputs.build_windows_amdgpu_families }}
-      test_linux_amdgpu_families:
-        description: Linux families to run tests for.
-        value: ${{ jobs.setup.outputs.test_linux_amdgpu_families }}
-      test_windows_amdgpu_families:
-        description: Windows families to run tests for.
-        value: ${{ jobs.setup.outputs.test_windows_amdgpu_families }}
+      linux_amdgpu_families:
+        description: Linux families to run builds and tests on.
+        value: ${{ jobs.setup.outputs.linux_amdgpu_families }}
+      run_linux_tests_only:
+        description: If enabled, CI will only run Linux tests
+        value: ${{ jobs.setup.outputs.run_linux_tests_only }}
```

Comment:
Same comment about naming here. Does "run linux tests only" also turn off Windows tests? The "use_prebuilt_artifacts" naming would avoid that confusion.